src/audio/google/google_rtc_audio_processing.c

```diff
@@ -424,15 +429,23 @@ static struct comp_dev *google_rtc_audio_processing_create(
 		goto fail;
 	bzero(cd->raw_mic_buffer, cd->num_frames * cd->num_capture_channels * sizeof(cd->raw_mic_buffer[0]));
 	cd->raw_mic_buffer_frame_index = 0;
+	for (channel = 0; channel < cd->num_capture_channels; channel++)
+		cd->raw_mic_buffer_ptrs[channel] = &cd->raw_mic_buffer[channel * cd->num_frames];
 
 	cd->aec_reference_buffer = rballoc(
 		0, SOF_MEM_CAPS_RAM,
 		cd->num_frames * sizeof(cd->aec_reference_buffer[0]) *
 		cd->num_aec_reference_channels);
 	if (!cd->aec_reference_buffer)
 		goto fail;
-	bzero(cd->aec_reference_buffer, cd->num_frames * cd->num_aec_reference_channels * sizeof(cd->aec_reference_buffer[0]));
+	bzero(cd->aec_reference_buffer,
+	      cd->num_frames *
+	      cd->num_aec_reference_channels *
+	      sizeof(cd->aec_reference_buffer[0]));
```

Comment:
Style nitpick: this is a whitespace-only change, and it faked me out because I thought for a second because I was sure that this was changing the bzero() region without updating the size of the allocation.

IMHO the whole "write the size expression in the alloc and bzero steps" pattern is fragile.  I guess my aesthetics would say that if we're going to change the source code bytes that compute this number, we should fix it so the size is computed exactly once and used as an immutable value in both the alloc/clear.